labs/firepower-restapi-101/2.md

```diff
@@ -1,67 +1,56 @@
+The API Explorer automates the access token process, and you can log into the API Explorer using any FMC account, but you will only be able to perform the functions for which the account has permissions. For more information on setting permissions for user roles, see the [Firepower Management Center Configuration Guide](http://www.cisco.com/c/en/us/td/docs/security/firepower/610/configuration/guide/fpmc-config-guide-v61/logging_into_firepower_system.html).
 
+## Logging into FMC
 
-Make sure you have an account on your Firepower Management Center with the required permissions to perform the tasks you want to perform using the REST API.
+Detailed information about logging into FMC is documented in the [Firepower Management Center Configuration Guide](http://www.cisco.com/c/en/us/td/docs/security/firepower/610/configuration/guide/fpmc-config-guide-v61/logging_into_firepower_system.html).
 
-## Logging procedure
-*  Navigate to the following URL: https://management_center_IP_or_name/api/api-explorer or try [DevNet Always-On Sandbox](https://fmcrestapisandbox.cisco.com/api/api-explorer/):
-*  Log in with your username and password and click **Sign In**
+To log into the API Explorer, navigate to your FMC instance or an active FMC sandbox and log in with your username and password,
 
 ## Setting the domain
-The REST API can perform functions globally, in the default domain, or in specific domains.
-You must have a configured Firepower Management Center, and an account with permissions to access the REST API in one or more domains configured on your FMC.
-In the upper right, drop down the Domains menu and select the desired domain.
+
+The FMC REST API can perform functions globally, in the default domain, or in specific domains. To select the domain, you must have an account with permissions to access the REST API in one or more domains configured on your FMC. Select the desired domain from the **Domains** menu.
 
 ## Obtaining sample code
-You can use the Export Operation button in API Explorer to save the displayed method example as a Python or Perl script file to your local host. This script can be used to make that REST call, and can be integrated into your application. This is meant primarily as an educational and bootstrapping tool.
 
-Python: The Python scripts require you to install Python, available from https://www.python.org/. Once you’ve installed Python, you can run your script with python script.py username password.
-Perl: Using the Perl scripts requires some additional set-up—you need five components:
+The API Explorer can generate sample REST call code in Python and Perl. This code can be adapted to your specific inmplementation.
+
+1. In the **API INFO** menu, select a resource.
```

Comment:
select a resource, such as **Devices**.

Reply:
This seems to work on Mac on Chrome. You can ask Veer or check the 6.2.3 docs to see if that browser limitation changed. Then I can help you find the repo for that Sandbox tab content.